src/Sql/Sql/Sql.csproj

```diff
@@ -21,7 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.37.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.38.0-preview" />
```

Comment:
Azure sdk generation is currently blocked so 1.38.0 build doesn't actually exist.  However, I set it here so that once that sdk is generated, no changes have to be made here.